notebooks/MAST/astroqueryMAST/beginner_search_astroqueryMAST/beginner_search_astroqueryMAST.ipynb

```diff
@@ -0,0 +1,395 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "obsByCriteria = Observations.query_criteria(obs_collection=[\"HLA\"], s_dec=[50,60], \n",
+    "                                            calib_level=[3], proposal_pi=\"Mould*\", \n",
+    "                                            dataproduct_type=\"IMAGE\", t_max=[49800,49820])\n",
+    "\n",
+    "#obsByCriteria = Observations.query_criteria(obs_id = 'hst_05766_04_wfpc2_total_wf')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "obsByCriteria.show_in_notebook(display_length=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting Associated Data Products"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Performing a Product Query\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each observation returned from a MAST query can have one or more associated data products. When you input observations or observation ids (“obs_id”), `get_product_list` will return a table containing the associated data products. The product fields are documented [here](https://mast.stsci.edu/api/v0/_productsfields.html).\n",
+    "\n",
+    "Since we already have a list of observations, we can use that as the starting point for our query. To keep it simple, let's look at only the last observation from our search above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's select a small subset from our critera search above\n",
+    "newObsList = obsByCriteria[3:]\n",
+    "\n",
+    "# Now we get the list of products associated with that observation\n",
+    "dataProducts = Observations.get_product_list(newObsList)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "dataProducts.show_in_notebook(display_length=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Filtering the Data Products\n",
+    "\n",
+    "After the data products have been retrieved, you can use `filter_products` to download only data products that meet your given criteria. Available filters are listed [here](https://mast.stsci.edu/api/v0/_productsfields.html). Some examples are: “mrp_only” (Minimum Recommended Products) and “extension” (file extension).\n",
+    "\n",
+    "Note: the ‘AND’ operation is performed for a list of filters, and the ‘OR’ operation is performed within a filter set."
```

Comment:
I know this is copied from the astroquery.mast readthedocs, but I still find it confusing (and I think there might be an error in that section of RTD anyways).  I think the main point is that each criteria listed in the call to filter_products() is joined with an AND, and each option within a criteria is joined with an OR.  For example, filter_products(productType=['SCIENCE'], extensions=['fits', 'txt']) would only retrieve products that are SCIENCE and .fits file extensions OR SCIENCE and .txt file extensions.  Maybe this (or something similar) can be used to really drive home the description?